app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,30 @@
 <!DOCTYPE html>
```

Comment:
@bgdeutsch I would use a gem for the font awesome stuff to keep it consistent. 
https://github.com/FortAwesome/font-awesome-sass